content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -477,6 +477,12 @@ In the CLI, the access modes are abbreviated to:
 * RWX - ReadWriteMany
 * RWOP - ReadWriteOncePod
 
+{{< note >}}
+The access modes are used to match PersistentVolume and PersistentVolumeClaim.
+The access modes don't set any constraints on the volume.
```

Comment:
While it's true for some cases (`ReadOnlyMany` mentioned below), it is actually enforced for `ReadWriteOncePod`.

Reply:
@jsafrane 
Thank you for your review and comment.
I have updated the explanation.
Can you please check it.